packages/manager/modules/product-offers/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ovh-ux/manager-product-offers",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Set of components and services to handle option order with different workflows",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/product-offers"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVHcloud SAS",
+  "main": "./src/index.js",
+  "dependencies": {
+    "lodash": "^4.17.15",
+    "moment": "^2.24.0"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^7.6.2",
+    "@ovh-ux/ng-ovh-contracts": "^3.1.1",
+    "@ovh-ux/ng-ovh-payment-method": "^5.2.0",
+    "@ovh-ux/ng-ovh-web-universe-components": "^6.0.1",
+    "@ovh-ux/ng-translate-async-loader": "^2.0.3",
+    "angular": "^1.7.9",
+    "angular-translate": "^2.18.2",
+    "bootstrap4": "^0.0.1-security",
```

Comment:
why not use the latest version 4.1.3?

Reply:
Copy paste

Reply:
I will use `twbs/bootstrap#v4.0.0` to avoid multiple versions on yarn.lock